src/components/QuestionAnswer.js

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from 'react';
+import { subscribe } from './Event';
+import { getQuestion } from './Helper';
+
+function QuestionAnswer(props) {
+  const { questionNum, nextQuestion } = props;
+  const [questionObj, setQuestion] = useState(getQuestion(questionNum));
+  const [inputVal, setInputVal] = useState('');
+  useEffect(() => {
+    subscribe('NextQuestion', () => {
+      nextQuestion(questionObj);
+      setQuestion(getQuestion(questionNum + 1));
+      setInputVal('');
+    });
+  });
+  const handleOnClick = () => {
+    nextQuestion(questionObj);
+    setQuestion(getQuestion(questionNum + 1));
+    setInputVal('');
+  };
+  const onChangeHandler = (e) => {
+    setQuestion((prevState) => ({
+      question: prevState.question,
+      id: prevState.id,
+      answer: prevState.answer,
+      givenAnswer: e.target.value,
+      correct: Number(prevState.answer) === Number(e.target.value),
+    }));
+    setInputVal(e.target.value);
+  };
+  return (
```

Comment:
Also add a functionaliy here. User can also press enter instead of clicking Next

Reply:
Done